patches/kex_rlwe_newhope_avx2/patch1.patch

```diff
@@ -0,0 +1,1073 @@
+diff --git a/src/kex_rlwe_newhope/avx2/kex_rlwe_newhope_avx2.c b/src/kex_rlwe_newhope/avx2/kex_rlwe_newhope_avx2.c
+new file mode 100644
+index 0000000..5e750c3
+--- /dev/null
++++ b/src/kex_rlwe_newhope/avx2/kex_rlwe_newhope_avx2.c
+@@ -0,0 +1,159 @@
++#if defined(WINDOWS)
++#define UNUSED
++// __attribute__ not supported in VS, is there something else I should define?
++#else
++#define UNUSED __attribute__((unused))
++#endif
++
++#include <stdlib.h>
++#include <string.h>
++#if !defined(WINDOWS)
++#include <strings.h>
++#include <unistd.h>
++#endif
++
++#include <oqs/kex.h>
++#include <oqs/rand.h>
++
++#include "kex_rlwe_newhope_avx2.h"
++#include "newhope.c"
++#include "params.h"
++
++OQS_KEX *OQS_KEX_rlwe_newhope_avx2_new(OQS_RAND *rand) {
```

Comment:
Should this be in the patch?

Reply:
I meant, should these two files be in the patch?

Reply:
Removed the binary files from under test directory.
All the files and changes in the patch should be there because the aim is to keep the external original code untouched as much as possible.